lib/const.rb

```diff
@@ -2,18 +2,25 @@
 # Defines common constants used in different parts of crew
 require 'etc'
 
-CREW_VERSION = '1.49.3'
+CREW_VERSION = '1.49.4'
 
 # Kernel architecture.
 KERN_ARCH = Etc.uname[:machine]
 
 # Read and parse processor information from /proc/cpuinfo
 CPUINFO = File.readlines('/proc/cpuinfo').map { |line| line.chomp.split(/\t+: /) if line.include?("\t") }.compact.to_h
 
-# We report aarch64 machines as armv7l for now, as we treat them as if they were armv7l.
+# We report aarch64 & armv8l machines as armv7l for now, as we treat
```

Comment:
There are no chromeos `armv8l` machines, so at the very least mention that this is a container-specific workaround when running the containers on certain hardware.

Reply:
armv8l is just another way to say aarch64.

Reply:
No, it is not. No chromeos machines report `armv8l`.

Reply:
The point is Docker containers do report armv8l.  We can't just remove logic that isn't related to ChromeOS devices.

Reply:
Docker containers don't report it, raspberry pi's report it. 

I don't think this logic should be located as part of crew, i think it makes more sense to have it as part of docker, because it only affects docker containers.